compiler/hash-layout/src/lib.rs

```diff
@@ -56,18 +62,25 @@ pub struct PointeeInfo {
     pub kind: Option<PointerKind>,
 }
 
-// Define a new key to represent a particular layout.
-new_store_key!(pub LayoutId, derives = Debug);
+stores!(
+    LayoutStores;
+    layouts: LayoutStore
+);
+
+/// The global [`LayoutStores`] instance.
+static STORES: OnceLock<LayoutStores> = OnceLock::new();
+
+/// Access the global [`LayoutStores`] instance.
+pub(crate) fn layout_store() -> &'static LayoutStores {
+    STORES.get_or_init(LayoutStores::new)
+}
```

Comment:
Might as well suffix with `.layouts()` since that's the only store?

Reply:
Do you mean to just call it `Layouts`?

Reply:
See suggestion below